modules/Kconfig.mcuboot

```diff
@@ -256,6 +256,16 @@ config MCUBOOT_BOOTLOADER_NO_DOWNGRADE
 	  MCUBOOT_DOWNGRADE_PREVENTION option enabled.
 endif
 
+config MCUBOOT_APPLICATION_FIRMWARE_UPDATER
+	bool "Application is firmware updater image"
+	depends on MCUBOOT_BOOTLOADER_MODE_FIRMWARE_UPDATER
+	help
+	  Select this if the current image is the firmware updater image. This will use slot 1
+	  information when signing the image.
+
+	  Note that the zephyr chosen node ``zephyr,code-partition`` should be set to
+	  ``slot1_partition`` for this image.
```

Comment:
if `this image` is the image currently being built, then isn't it possible to add a dependency to the devicetree chosen node ?

Reply:
discussed offline.
I believe the help text can be improved, but that is just nice to have.